webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -240,10 +325,19 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
           agentUserGroup?: V1AgentUserGroup;
           checkpointStorageConfig?: unknown;
           name: string;
+          clusterNamespacePairs?: { [key: string]: string };
         } = {
           name: workspaceName,
         };
 
+        const clusterNamespacePairs: { [key: string]: string } = {};
+        resourceManagers.map((name) => {
+          if (name in values && values[name] !== undefined) {
+            clusterNamespacePairs[name] = '' + values[name];
+          }
+        });
```

Comment:
please use `reduce` here:
```suggestion
        const clusterNamespacePairs: = resourceManagers.reduce((memo: Record<string>, name) => {
          if (name in values) {
            memo[name] = values[name].toString();
          }
          return memo
        }, {});
```

Reply:
done!